frontend/admin-ui/package.json

```diff
@@ -14,6 +14,7 @@
     "apollo-client": "^2.6.4",
     "apollo-link": "^1.2.13",
     "apollo-link-http": "^1.5.16",
+    "core-js": "^2.6.5",
```

Comment:
If that package is not registered here, then I can't build the admin-ui with lerna.